fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -1330,14 +1330,21 @@ private void updateLoanSummaryDerivedFields() {
             this.totalOverpaid = null;
         } else {
             final Money overpaidBy = calculateTotalOverpayment();
-            this.totalOverpaid = overpaidBy.getAmountDefaultedToNullIfZero();
+            this.totalOverpaid = null;
+            Money credits = Money.zero(getCurrency());
+            Money debits = Money.zero(getCurrency());
```

Comment:
Debits is always zero?

Reply:
For now yes, there is not any to be consider as debit